### PR TITLE
add pagination

### PR DESCRIPTION
**Pull request by whiteadam2**

### CODE REVIEW

File: apps/monitor-client/lib/features/users-view/components/UsersView.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+export function UsersView() {
+    const {
+        usersData,
+        isUsersLoading,
+        isUsersError,
+        usersPage,
+        setUsersPage,
+        usersPageSize,
+        setUsersPageSize,
```
Comment (yzhbankov) [inline apps/monitor-client/lib/features/users-view/components/UsersView.tsx:22]: I think we can change entity-centric names to common:
const { data, isLoading, isError, page, setPage, pageSize, setPageSize } = useFetchUsers();

The same approach is for other hooks. And this changes should be applied for hook itself

Comment (whiteadam2) [reply apps/monitor-client/lib/features/users-view/components/UsersView.tsx:22]: but how we should manage data from different hooks among one component. we should somehow recognize data is got from different hooks

File: apps/monitor-client/lib/index.tsx [hunk 2]
```diff
@@ -6,15 +6,20 @@ import { AppProvider } from './context';
 import { ErrorBoundaryComponent } from './components/ErrorBoundary';
 import './index.css';
 
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+const queryClient = new QueryClient();
```
Comment (yzhbankov) [inline apps/monitor-client/lib/index.tsx:11]: lets move queryClient to the separate file (./lib/queryClient/index.ts) and pass global config to play with it:
export const queryClient = new QueryClient({
  defaultOptions: {
    queries: {
      staleTime: 60 * 1000 // 1 minutes,
      cacheTime: 10 * 60 * 1000 // 10 minutes
    }
  }
})

File: apps/monitor-client/lib/features/users-view/components/UsersView.tsx (hunk 1 above)
Comment (yzhbankov) [reply apps/monitor-client/lib/features/users-view/components/UsersView.tsx:22]: do we have such a component?

Comment (yzhbankov) [reply apps/monitor-client/lib/features/users-view/components/UsersView.tsx:22]: even if we have this will be resolved by the import

**Pull request merged**